corehq/apps/reports/util.py

```diff
@@ -384,13 +384,17 @@ def is_query_too_big(domain, mobile_user_and_group_slugs, request_user):
     return user_es_query.count() > USER_QUERY_LIMIT
 
 
-def send_report_download_email(title, recipient, link, subject=None, domain=None):
+def send_report_download_email(title, recipient, link, subject=None, domain=None, limit=None):
     if subject is None:
         subject = _("%s: Requested export excel data") % title
     body = "The export you requested for the '%s' report is ready.<br>" \
            "You can download the data at the following link: %s<br><br>" \
            "Please remember that this link will only be active for 24 hours."
 
+    if limit:
```

Comment:
I played around with passing back a flag so this message could be added only if the file is genuinely too big and ended up deciding the complexity wasn't worth it. I could open the actual file in `send_report_download_email` and check it, but that seems potentially slow/intensive for large files.